docs/source/new-features/index.rst

```diff
@@ -9,6 +9,78 @@ Find out about the details of major features, changes, and deprecations below.
 
     <section>
         <article class="new-content-grid">
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        MLflow Site Overhaul 
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
```

Comment:
Can we add screenshot for the new home page?

Reply:
Great idea! Added!